src/test/java/org/fundacionjala/coding/david/EANValidationTest.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.david;
+import org.junit.Test;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This class is for test check number.
+ */
+public class EANValidationTest {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(eanValidation.checksumValidation("9783827317100"));
```